<?xml version='1.0' encoding='UTF-8'?>
<rss xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/" version="2.0">
  <channel>
    <title>CVE Feed for PrestaShop -- ps_socialfollow</title>
    <link>https://raw.githubusercontent.com/deepseas/cvelistV5monitor/main/feeds/prestashop/ps_socialfollow.rss</link>
    <description>The latest CVEs for PrestaShop -- ps_socialfollow</description>
    <docs>http://www.rssboard.org/rss-specification</docs>
    <generator>python-feedgen</generator>
    <lastBuildDate>Sun, 04 Aug 2024 18:24:36 +0000</lastBuildDate>
    <ttl>60</ttl>
    <item>
      <title>CVE-2020-5294|2024-08-04T08:22:09.096Z -- prestashop -- ps_socialfollow
</title>
      <link>https://www.cve.org/CVERecord?id=CVE-2020-5294</link>
      <description>PrestaShop module ps_facetedsearch versions before 2.1.0 has a reflected XSS with social networks fields The problem is fixed in 2.1.0</description>
      <guid isPermaLink="false">CVE-2020-5294|2024-08-04T08:22:09.096Z</guid>
      <pubDate>Thu, 16 Apr 2020 21:15:13 +0000</pubDate>
    </item>
  </channel>
</rss>
